### DISCUSSION TITLE
Is it possible to prevent page scrolling with useWheel(react)?

### DISCUSSION THREAD

**Discussion by hellozyemlya** [pmndrs/use-gesture#417] [Q&A] [answered]
Question is self-explanatory. I want detect wheel event but leave page(and possible nested components) scrollbars untouched.

Thanks in advance.

PS: nice library for lazy persons like me, it includes everything we need :)

**Comment by dbismut** [comment, marked as answer]
Wheel event is independent from scroll. If you have a fixed position body with no overflow, your page won't scroll!

**Reply by hellozyemlya** [reply]
Roger that, going to make changes to my page.
